src/platform/meteorlake/include/platform/lib/ams.h

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2023 Intel Corporation. All rights reserved.
+ */
+
+#ifdef __SOF_LIB_AMS_H__
+
+#ifndef __PLATFORM_LIB_AMS_H__
+#define __PLATFORM_LIB_AMS_H__
+
+/* max number of message UUIDs */
+#define AMS_SERVICE_UUID_TABLE_SIZE 16
```

Comment:
Can you give us a sense of where this 16 comes from, e.g. how many are in use today and how much is left?

Reply:
Sure. This value is abstract at the moment. There is no usecase. There are features waiting for async messages, so that should come rather sooner than later. AFAIK async messages might/should replace notification system in the future, so the necessary sizes will be finetuned.

Reply:
I can't believe there is no known size here. If this AMS thingy has been tried and tested, surely there's a better answer than "this is what is is and will be modified later'.